### ISSUE TITLE
Can I predict with your pretrained model for Python 3.6 in Ubuntu, only with my CPU?

### ISSUE THREAD

**Issue by Jars-of-jam-Scheduler** [nagadomi/waifu2x#346]
Hello everyone, 

I don't have any GPU on my work station. So can I predict with your pretrained model for Python 3.6 in Ubuntu, only with my CPU, without installing NVIDIA software (CUDA, etc.)?

Thank you in advance!
Have a great day and stay safe,
Best regards,

**Comment by nagadomi** [comment]
PyTorch version can be available at https://github.com/nagadomi/nunif (only convert command is supported). However, CUDA is required.
https://github.com/nihui/waifu2x-ncnn-vulkan supports Intel/AMD/NVIDIA.
